From: "Laszlo Boszormenyi (GCS)" <gcs@debian.org>
Date: Sun, 24 Nov 2024 23:29:28 -0800
Subject: do not build tools with rpath

Forwarded: not-needed
Last-Update: 2021-10-16

These will be installed semi-manual to usr/bin and will use system libraries.
---
 CMakeLists.txt       | 1 +
 tools/CMakeLists.txt | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f1103c5..b17f9b5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1568,6 +1568,7 @@ if(WITH_TESTS)
 endif()
 
 if(WITH_BENCHMARK_TOOLS)
+  set(CMAKE_SKIP_BUILD_RPATH TRUE)
   add_executable(db_bench${ARTIFACT_SUFFIX}
     tools/simulated_hybrid_file_system.cc
     tools/db_bench.cc
diff --git a/tools/CMakeLists.txt b/tools/CMakeLists.txt
index 19030e8..a66410d 100644
--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -1,3 +1,5 @@
+set(CMAKE_SKIP_BUILD_RPATH TRUE)
+
 set(CORE_TOOLS
   sst_dump.cc
   ldb.cc)
